html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
Depending on the outcome of the spec discussion, this might belong in a CSS spec folder.

Reply:
Moved to css-text-decor.